### PR TITLE
Task#5 - Feat: Create navbar section

### LINKED ISSUE #14: DevTask - Create section components
- [x] #15 
- [x] #17 
- [x] #19
- [x] #21 

After fix reponsive for all section
- [x] #32

### LINKED ISSUE #15: Task#5 - DevTask: Create navbar section
## OVERVIEW
- Create section components: navbar
   -  logo icon
   -  navigation
   - button
- Responsive
- Testing
### DEMO
Topbar

### PR DESCRIPTION
**Pull request by TheHienDuong**
- Closes #15 
## OVERVIEW
- Remove content test in App.jsx
- Import navbar
- Edit button-primary and button-secondary
- Create Navbar component
- Install logo Icon
- Add font Family

## Demo

https://github.com/Agility-Internship/Duong-The-Hien-ReactJS/assets/113186169/ee62396b-c709-4e40-9796-49d2344fb838

### CODE REVIEW

File: practice-one/src/Components/Section/Navbar.jsx [hunk 1]
```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+import Button from '../Common/Buttons';
+
+const Navbar = () => (
+  <div className=" bg-light">
```
Comment (phattruong-agilityio) [inline practice-one/src/Components/Section/Navbar.jsx]: code styles

File: practice-one/src/Components/Section/Navbar.jsx [hunk 2]
```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+import Button from '../Common/Buttons';
+
+const Navbar = () => (
+  <div className=" bg-light">
+    <nav className="container mx-auto w-screen h-[80px]  ">
```
Comment (phattruong-agilityio) [inline practice-one/src/Components/Section/Navbar.jsx]: code styles

File: practice-one/src/Components/Section/Navbar.jsx [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+const Navbar = () => (
+  <div className=" bg-light">
+    <nav className="container mx-auto w-screen h-[80px]  ">
+      <div className="px-2 flex justify-between items-center w-full h-full">
+        <div className="flex items-center gap-5">
+          <a className="navbar-brand logo" href="#">
+            <img src="src\assets\logo.png" alt="logo" />
+          </a>
+          <ul className="hidden ps-20 gap-10  md:flex">
+            <li><a href="/Product" className="text-base hover:text-primary">Product</a></li>
```
Comment (phattruong-agilityio) [inline practice-one/src/Components/Section/Navbar.jsx]: Check the href link, It's should be #product

File: practice-one/src/Components/Section/Navbar.jsx [hunk 4]
```diff
@@ -0,0 +1,37 @@
...
+            <li><a href="/Contact" className="text-base hover:text-primary">Pricing</a></li>
```
Comment (phattruong-agilityio) [inline practice-one/src/Components/Section/Navbar.jsx]: Check the href link, It's should be #contact

File: practice-one/src/Components/Section/Navbar.jsx [hunk 5]
```diff
@@ -0,0 +1,37 @@
...
+            <li><a href="/Company" className="text-base hover:text-primary">Company</a></li>
```
Comment (phattruong-agilityio) [inline practice-one/src/Components/Section/Navbar.jsx]: Check the href link, It's should be #company

File: practice-one/src/Components/Section/Navbar.jsx [hunk 6]
```diff
@@ -0,0 +1,37 @@
...
+            <li><a href="/Recourses" className="text-base hover:text-primary">Recourses</a></li>
```
Comment (phattruong-agilityio) [inline practice-one/src/Components/Section/Navbar.jsx]: Check the href link, It's should be #recourses

File: practice-one/src/Components/Section/Navbar.jsx [hunk 7]
```diff
@@ -0,0 +1,37 @@
...
+            <li><a href="/Contact" className="text-base hover:text-primary">Contact</a></li>
```
Comment (phattruong-agilityio) [inline practice-one/src/Components/Section/Navbar.jsx]: Check the href link, It's should be #contact

File: practice-one/src/Components/Section/Navbar.jsx [hunk 8]
```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+import Button from '../Common/Buttons';
```
Comment (phattruong-agilityio) [inline practice-one/src/Components/Section/Navbar.jsx]: it should be ../Common/Button not ../Common/Buttons

File: practice-one/src/Components/Section/Navbar.jsx [hunk 9]
```diff
@@ -0,0 +1,37 @@
...
+
+const Navbar = () => (
+  <div className=" bg-light">
+    <nav className="container mx-auto w-screen h-[80px]  ">
+      <div className="px-2 flex justify-between items-center w-full h-full">
+        <div className="flex items-center gap-5">
+          <a className="navbar-brand logo" href="#">
+            <img src="src\assets\logo.png" alt="logo" />
+          </a>
+          <ul className="hidden ps-20 gap-10  md:flex">
```
Comment (phattruong-agilityio) [inline practice-one/src/Components/Section/Navbar.jsx]: code style

File: practice-one/src/App.jsx [hunk 10]
```diff
@@ -1,59 +1,8 @@
 import React from 'react';
-import ProductCard from './Components/Common/productCard';
-import ServiceCard from './Components/Common/serviceCard';
-import HorizontalCard from './Components/Common/horizontalCard';
+import Navbar from './Components/Section/Navbar';
```
Comment (phattruong-agilityio) [inline practice-one/src/App.jsx:3]: Please add comment 
// Components  
before import

File: practice-one/src/Components/Section/Navbar.jsx [hunk 11]
```diff
@@ -0,0 +1,49 @@
...
+            <li>
+              <a href="/Product" className="text-base hover:text-primary">
```
Comment (phattruong-agilityio) [inline practice-one/src/Components/Section/Navbar.jsx]: Why don't use component Nav.Link

File: practice-one/src/Components/Section/Navbar.jsx [hunk 12]
```diff
@@ -0,0 +1,49 @@
...
+            </li>
+            <li>
+              <a href="/Contact" className="text-base hover:text-primary">
+                Contact
+              </a>
+            </li>
+          </ul>
+        </div>
+        <div className="hidden md:flex pr-4 gap-5">
+          <Button text="Log in" size="medium" variant="primary" />
```
Comment (phattruong-agilityio) [inline practice-one/src/Components/Section/Navbar.jsx]: size "medium" seem a default value
The same with variant

File: practice-one/src/Components/Section/Navbar.jsx (hunk 1 above)
Comment (TheHienDuong) [reply practice-one/src/Components/Section/Navbar.jsx]: Done

File: practice-one/src/Components/Section/Navbar.jsx (hunk 2 above)
Comment (TheHienDuong) [reply practice-one/src/Components/Section/Navbar.jsx]: Done

File: practice-one/src/Components/Section/Navbar.jsx (hunk 3 above)
Comment (TheHienDuong) [reply practice-one/src/Components/Section/Navbar.jsx]: Checked and fixed

File: practice-one/src/Components/Section/Navbar.jsx (hunk 4 above)
Comment (TheHienDuong) [reply practice-one/src/Components/Section/Navbar.jsx]: Checked and fixed

File: practice-one/src/Components/Section/Navbar.jsx (hunk 5 above)
Comment (TheHienDuong) [reply practice-one/src/Components/Section/Navbar.jsx]: Checked and fixed

File: practice-one/src/Components/Section/Navbar.jsx (hunk 6 above)
Comment (TheHienDuong) [reply practice-one/src/Components/Section/Navbar.jsx]: Checked and fixed

File: practice-one/src/Components/Section/Navbar.jsx (hunk 7 above)
Comment (TheHienDuong) [reply practice-one/src/Components/Section/Navbar.jsx]: Checked and fixed

File: practice-one/src/Components/Section/Navbar.jsx (hunk 8 above)
Comment (TheHienDuong) [reply practice-one/src/Components/Section/Navbar.jsx]: Thanks! Change file Button name and update link name

File: practice-one/src/Components/Section/Navbar.jsx (hunk 9 above)
Comment (TheHienDuong) [reply practice-one/src/Components/Section/Navbar.jsx]: Done

File: practice-one/src/App.jsx (hunk 10 above)
Comment (TheHienDuong) [reply practice-one/src/App.jsx:3]: Thanks, Added comment before imports

File: practice-one/src/Components/Section/Navbar.jsx (hunk 11 above)
Comment (TheHienDuong) [reply practice-one/src/Components/Section/Navbar.jsx]: Do you mean to use Nav.link from another library? React-Bootstrap example?

File: practice-one/src/Components/Section/Navbar.jsx (hunk 12 above)
Comment (TheHienDuong) [reply practice-one/src/Components/Section/Navbar.jsx]: Thanks, I understood the problem, taked note and updated the button size

File: practice-one/package.json [hunk 13]
```diff
@@ -19,6 +19,7 @@
         "@vitejs/plugin-react-swc": "^3.3.2",
         "prop-types": "^15.8.1",
         "react": "^18.2.0",
+        "react-bootstrap": "^2.8.0",
```
Comment (phattruong-agilityio) [inline practice-one/package.json]: What is this? Does this one have in tech-stack?

File: practice-one/src/Components/Section/Navbar.jsx [hunk 14]
```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+import { Nav } from 'react-bootstrap';
```
Comment (phattruong-agilityio) [inline practice-one/src/Components/Section/Navbar.jsx]: What is this?

File: practice-one/src/Components/Section/Navbar.jsx [hunk 15]
```diff
@@ -0,0 +1,70 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import Button from '../Common/Button';
```
Comment (phattruong-agilityio) [inline practice-one/src/Components/Section/Navbar.jsx:4]: Missing comment // Components here

Comment (TheHienDuong) [reply practice-one/src/Components/Section/Navbar.jsx:4]: Done

**Pull request merged**